var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -17,8 +17,8 @@ class Charmpp(Package):
     allows programs to run portably from small multicore computers
     (your laptop) to the largest supercomputers."""
 
-    homepage = "http://charmplusplus.org"
-    url      = "http://charm.cs.illinois.edu/distrib/charm-6.8.2.tar.gz"
+    homepage = "https://charmplusplus.org"
+    url      = "https://charm.cs.illinois.edu/distrib/charm-6.8.2.tar.gz"
     git      = "https://github.com/UIUC-PPL/charm.git"
 
     version("develop", branch="master")
```

Comment:
```suggestion
    version("master", branch="master")
```

Reply:
By the way, is it possible to specify an arbitrary git branch on the command line? Something like `spack install charmpp@foobar`?

Reply:
Nope, not yet